tests/framework/CodeInterfaceTests/test_Dymola_code_interface_load_some_vars.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" ?>
+<Simulation>
+  <TestInfo>
+    <name>framework/CodeInterfaceTests.DymolaTestLoadSomeVarsNoExecutable</name>
+    <author>@alfoa</author>
+    <created>2017-09-16</created>
+    <classesTested>CodeInterfaces.Dymola</classesTested>
+    <description>
+       The purpose of this test is to exercise the Dymola external code interface without needing to have a built 
+         executable.  This special case shows how to load just few output varibles from the Dymola outputs (e.g. .mat files).
+         In this case only the variables e,g and h will be loaded
+    </description>
+  </TestInfo>
+  <Files>
+    <Input name="dsin.txt" type="DymolaInitialisation">dsin.txt</Input>
+  </Files>
+  <RunInfo>
+    <WorkingDir>DMInterfaceTestLoadSomeVars</WorkingDir>
+    <Sequence>testDummyStep</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Models>
+    <Code name="BouncingBall" subType="Dymola">
+      <executable>
+        <!-- here we should put the executable. Since we want to test the interface only, it is going to be empty -->
+      </executable>
+      <outputVariablesToLoad>e,g,h</outputVariablesToLoad>
```

Comment:
The syntax looks OK. Just a question:
Could you have used the alas system to find the outputs? Most codes I used have definitions like:
 <alias variable='RAVEN name' type='output'>DYMOLA_name</alias>

Reply:
the alias system is handled directly at RAVEN master level.
It is intended to change the variable names but not to find them :(